packages/playground/storage/src/lib/git-sparse-checkout.ts

```
@@ -67,14 +92,25 @@ export async function sparseCheckout(
 	filesPaths: string[],
 	options?: {
 		withObjects?: boolean;
+		additionalHeaders?: GitAdditionalHeaders;
 	}
 ): Promise<SparseCheckoutResult> {
-	const treesPack = await fetchWithoutBlobs(repoUrl, commitHash);
+	const treesPack = await fetchWithoutBlobs(
+		repoUrl,
+		commitHash,
+		resolveGitHeaders(repoUrl, options?.additionalHeaders)
```

@akirk Do we need a custom function for that? Wouldn't `options?.additionalHeaders?.(repoUrl)` be enough? As a bonus point, it would throw an error if the type is not a function (right now we silently skip that).

---

Indeed, we don't need it. I didn't clean up some initially more complex code. Thanks!